<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VistaPrincipal</class>
 <widget class="QMainWindow" name="VistaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QAxWidget" name="VisualizadorPdf">
      <property name="control" stdset="0">
       <string>{ca8a9780-280d-11cf-a24d-444553540000}</string>
      </property>
      <property name="sizePolicy" stdset="0">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation" stdset="0">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>643</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSistema">
    <property name="title">
     <string>Sistema</string>
    </property>
    <addaction name="separator"/>
    <addaction name="opcionSalir"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
   </widget>
   <widget class="QMenu" name="menuHerramientas">
    <property name="title">
     <string>Herramientas</string>
    </property>
    <addaction name="actionCopiar"/>
    <addaction name="actionCortar"/>
    <addaction name="actionPegar"/>
    <addaction name="separator"/>
    <addaction name="actionImprimir"/>
    <addaction name="actionExportar"/>
    <addaction name="actionDigitalizar"/>
    <addaction name="separator"/>
    <addaction name="actionCargar_Archivos"/>
   </widget>
   <addaction name="menuSistema"/>
   <addaction name="menuHerramientas"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionImprimir"/>
   <addaction name="actionDigitalizar"/>
   <addaction name="actionExportar"/>
   <addaction name="separator"/>
   <addaction name="actionCortar"/>
   <addaction name="actionCopiar"/>
   <addaction name="actionPegar"/>
   <addaction name="separator"/>
   <addaction name="actionAcerca_de"/>
  </widget>
  <widget class="QDockWidget" name="Buscador">
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Tipo Documental</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTreeView" name="treeViewBuscador"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="Llenado">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Llenado</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_11">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QTableWidget" name="tableWidget_Llenado"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="opcionUsuarios">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>archivos/src/img/grupo.png</normaloff>archivos/src/img/grupo.png</iconset>
   </property>
   <property name="text">
    <string>Usuarios</string>
   </property>
  </action>
  <action name="opcionSalir">
   <property name="icon">
    <iconset>
     <normaloff>../src/img/cancelar.png</normaloff>../src/img/cancelar.png</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionCopiar">
   <property name="icon">
    <iconset>
     <normaloff>../src/img/copiar.png</normaloff>../src/img/copiar.png</iconset>
   </property>
   <property name="text">
    <string>Copiar</string>
   </property>
  </action>
  <action name="actionCortar">
   <property name="icon">
    <iconset>
     <normaloff>../src/img/tijeras.png</normaloff>../src/img/tijeras.png</iconset>
   </property>
   <property name="text">
    <string>Cortar</string>
   </property>
  </action>
  <action name="actionPegar">
   <property name="icon">
    <iconset>
     <normaloff>../src/img/pegar.png</normaloff>../src/img/pegar.png</iconset>
   </property>
   <property name="text">
    <string>Pegar</string>
   </property>
  </action>
  <action name="actionImprimir">
   <property name="icon">
    <iconset>
     <normaloff>../src/img/impresora.png</normaloff>../src/img/impresora.png</iconset>
   </property>
   <property name="text">
    <string>Imprimir</string>
   </property>
  </action>
  <action name="actionExportar">
   <property name="icon">
    <iconset>
     <normaloff>../src/img/exportar.png</normaloff>../src/img/exportar.png</iconset>
   </property>
   <property name="text">
    <string>Exportar</string>
   </property>
  </action>
  <action name="actionDigitalizar">
   <property name="icon">
    <iconset>
     <normaloff>../src/img/tableta-digitalizadora.png</normaloff>../src/img/tableta-digitalizadora.png</iconset>
   </property>
   <property name="text">
    <string>Digitalizar</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="icon">
    <iconset>
     <normaloff>../src/img/ayuda.png</normaloff>../src/img/ayuda.png</iconset>
   </property>
   <property name="text">
    <string>Acerca de</string>
   </property>
  </action>
  <action name="actionCargar_Archivos">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cargar Archivos</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QAxWidget</class>
   <extends>QWidget</extends>
   <header>qaxwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
